src/shared/cloudformation/cloudformation.ts

```diff
@@ -823,9 +826,17 @@ export async function refreshSchemas(extensionContext: vscode.ExtensionContext):
 export async function updateYamlSchemasArray(
     path: string,
     type: 'cfn' | 'sam' | 'none',
-    config: WorkspaceConfiguration = vscode.workspace.getConfiguration('yaml'),
-    paths: { cfnSchema: string; samSchema: string } = { cfnSchema: CFN_SCHEMA_PATH, samSchema: SAM_SCHEMA_PATH }
+    testOverrides?: {
```

Comment:
Maybe just call this `options`? It's used for testing, though it doesn't necessarily need to be.